src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -313,4 +314,14 @@ public void setAdditionalArguments(List<String> additionalArguments) {
     this.additionalArguments =
         additionalArguments != null ? ImmutableList.copyOf(additionalArguments) : null;
   }
+
+  @Override
+  @Nullable
+  public String getProject() {
+    return project;
+  }
+
+  public void setProject(String project) {
```

Comment:
Is this really a project? Or is it a project name or ID or some such thing? What do user docs call it?

Reply:
The docs call it the "application". Now that I think about it, `setApplication` may make more sense. @loosebazooka